ngraph_bridge/ngraph_encapsulate_impl.cc

```diff
@@ -535,11 +535,11 @@ Status NGraphEncapsulateImpl::ParseNodeAttributes(
   return Status::OK();
 }
 
-Status NGraphEncapsulateImpl::CachePipelinedTensorIfNeeded(
+Status NGraphEncapsulateImpl::UpdatePipelinedTensorCache(
```

Comment:
Rename to InitializePipelinedTensorCache ?

Reply:
`UpdatePipelinedTensorCache` was the name decided in one of the meetings, so used that.